app/templates/components/forms/admin/content/pages-form.hbs

```diff
@@ -8,7 +8,7 @@
     {{input type='text' name='title' value=data.title}}
   </div>
   <div class="field">
-    <label>{{t 'URL'}}</label>
+    <label>{{t 'URL (URL should be in the form \'privacy\', \'help\', \'terms/new\', \'help/contact\')'}}</label>
```

Comment:
Add a form validation check instead. Call this field as path instead of url.

Reply:
@niranjan94 I am using the regex ```[^/](.*)``` which is working fine in any online regex checker, but not in our form validation rules? Am I missing something?